.github/workflows/typedocs.yml

```diff
@@ -0,0 +1,24 @@
+name: Run commands when issues or PRs are labeled
+on:
+  issues:
+    types: [ labeled ]
+  pull_request:
+    types: [ labeled ]
+jobs:
+  main:
+    if: ${{ github.event.pull_request == false || github.event.pull_request.head.repo.full_name == github.repository }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v3
```

Comment:
https://github.com/grafana/plugin-tools/pull/395 we should probably use 4 as we've just bumped this in another action